### PR TITLE
Replace Roboto Font with Qanelas Soft Font

### LINKED ISSUE #8863: Replace Roboto Font with Qanelas Soft Font
### Current Behavior
Some sections of Meshery UI don't use the "Qanelas Soft" font

Example: 
image

### Desired Behavior
Audit the Meshery UI and update the font family to "Qanelas Soft" from "Roboto". Make sure that Open Sans and sans-serif are backup fonts.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by senali-d**
**Notes for Reviewers**

This PR fixes #8863

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Chadha93 — COMMENTED** [review]
@senali-d I think you missed pushing the font changes...

**Comment by senali-d** [comment]
> @senali-d I think you missed pushing the font changes...

@Chadha93 
What I have done here is instead of "@mui/material" Chip component import the @material-ui/core Chip.

**Reason:** 
We have already define font-family in custom theme file. So it is override @material-ui/core component fronts but not the "@mui/material".
So when use @material-ui/core component no need to override the component level styles. Theme file apply here and override the MUI styles.

**Comment by Chadha93** [comment]
> > @senali-d I think you missed pushing the font changes...
> 
> @Chadha93 What I have done here is instead of "@mui/material" Chip component import the @material-ui/core Chip.
> 
> **Reason:** We have already define font-family in custom theme file. So it is override @material-ui/core component fronts but not the "@mui/material". So when use @material-ui/core component no need to override the component level styles. Theme file apply here and override the MUI styles.

Right, I meant, will this change the font to all the components in Meshery UI? Yes, we've defined it in the theme. :heavy_check_mark:

**Comment by senali-d** [comment]
> 
> Right, I meant, will this change the font to all the components in Meshery UI? Yes, we've defined it in the theme. ✔️

Ok I'll check other components as well.

**Comment by jamieplu** [comment]
Aligning the project on the typography used in the design system will certainly give Meshery more polish.

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

**Comment by leecalcote** [comment]
merge conflicts

File: ui/components/configuratorComponents/MeshModel/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ import AppBarComponent from './styledComponents/AppBar';
 
 import DeleteIcon from '@material-ui/icons/Delete';
 import SaveIcon from '@material-ui/icons/Save';
-import { AvatarGroup } from '@mui/material';
+import { AvatarGroup } from '@material-ui/core';
```
Comment (sudhanshutech) [inline ui/components/configuratorComponents/MeshModel/index.js:18]: Is it ideal to use core elements from mui. Since we need to move to mui V5 then again we have to revert these changes

**Comment by senali-d** [comment]
Accidentally import MUI v5 components was fixed when we remove `MUI v5` package. Is it possible?? // @aabidsofi19 @nebula-aac 

Ref: [#6888](https://github.com/meshery/meshery/pull/6888)

**Comment by aabidsofi19** [comment]
Thanks @senali-d

**Comment by aabidsofi19** [comment]
After we remove all references to mui5 let's remove it from package.json

**Comment by senali-d** [comment]
> After we remove all references to mui5 let's remove it from package.json

@aabidsofi19 Meshery ui already use `MUI v5 ` icons. From the package `@mui/icons-material` so it depend on `@mui/material`. So we can't remove `MUI v5` from package.json.

Note: Some of the icons are newly introduced in `v5`

**Comment by leecalcote** [comment]
Merge conflict, @senali-d

**Comment by senali-d** [comment]
@leecalcote Fixed conflicts

**Comment by leecalcote** [comment]
@senali-d it could have been something else, but I seem to get this crash when running this branch:
Screenshot 2023-10-03 at 5 19 01 PM

**Comment by leecalcote** [comment]
@senali-d this item is in your court, I believe.

**Comment by senali-d** [comment]
> @senali-d this item is in your court, I believe.

Yup, I think build issue is not relate to changes in this pr. I'll open a new pr for this.

**Comment by senali-d** [comment]
Recently Meshery UI is Migrate from MUI V4 to MUI V5. Ref: #9319 
So I once done it no need these changes. Use of the new theme of MUI v5 fill fix this issue.

**Comment by leecalcote** [comment]
@senali-d please push this over the line.
